apps/roc_bindicator/roc_bindicator.star

```diff
@@ -0,0 +1,201 @@
+"""
+Applet: Roc Bindicator
+Summary: Show refuse in Rochester NY
+Description: Use address located in Rochester, NY to query the City website and then display what trash bins to but at the curb for collection.
+Author: Nolan Lynch
+"""
+load("render.star", "render")
+load("http.star", "http")
+load("schema.star", "schema")
+load("encoding/json.star", "json")
+
+DEFAULT_LOCATION = """
+{
+	"lat": "43.1529219",
+	"lng": "-77.6212444",
+	"description": "Rochester, NY, USA",
+	"locality": "Rochester",
+	"place_id": "ChIJCSF8lBZEwokRhngABHRcdoI",
+	"timezone": "America/New_York"
+}
+"""
+
+def main(config):
+
+#Get User Input Data from config
+    location = config.get("location", DEFAULT_LOCATION)
+    loc = json.decode(location)
+    Title_Texts = config.get("User_title_text", "Bindicator")
+    Bar_Color = config.get("bar_color", "#de00cb")
+    Positive_Color = config.get("Positive_Color", "#2f7000")
+    Negative_Color = config.get("negative_color", "#700000")
+
+#Convert config Location to Float from String
+    LatAsString = loc["lat"]
+    LatAsFloat = str(LatAsString)
+    LongAsString = loc["lng"]
+    LongAsFloat = str(LongAsString)
```

Comment:
How many decimals are needed in the lat-lng pair for the app to function properly? We try to make sure apps truncate them as far as possible to reduce the risk of unnecessarily sharing/leaking user's exact location with API providers.

Reply:
Due to pickup being based on your house address, the Lat/Long needs to be pretty specific. There are some neighborhoods where the left side of the street is a different date than the right side. So we need to pinpoint exactly where to query data for.

Reply:
Got it, thanks for the clarification!